frontend/src/Pages/SponsorUsPage/SponsorUsPage.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+function SponsorUsPage() {
+  return (
+    <div>
+      <h2 
+      style={{ 
+        textAlign: 'center', 
+        fontFamily: 'Mokoto',
+        fontWeight: '400',
+        color: 'white',
+        fontSize: '3rem',
+        marginTop: '50px'
+    }}
+    >
+      Sponsor Us!
+    </h2>
+      <div style={{
+        display: 'flex', 
+        flexDirection: 'row', 
+        justifyContent: 'center', 
+        alignItems: 'center',
+        columnGap: '10%', 
+        padding: '2% 10%'
+      }}>
+        <div style={{ 
+          width: '50%', 
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center'
+          }}
+          >
+          <h3 style={{ 
+            textAlign: 'center'
+            }}
+            >
+              Want to support us?
```

Comment:
Can you change the font here to be Poppins? It comes from the `fonts.css` so you can specify it like that.

Reply:
I changed it!